### PR TITLE
feat: Enable bulk hosts addition/removal to/from groups

### PR DESCRIPTION
**Pull request by gkarat**
Implements https://issues.redhat.com/browse/ESSNTL-3731, https://issues.redhat.com/browse/ESSNTL-4197.

This makes it possible to select more hosts and remove or add the to groups.

## How to test

1. Navigate to the inventory table
2. Select more hosts which are not in any group
3. Try to add them to existing group via the Add to group action. In the end you should see the failing network request with 405 status code - it is expected for now.
4. Try to remove hosts from the same group via the Remove from group action. The endpoint is implemented and the operation must be successful. Check the hosts are truly removed from the group.

## Screenshots

image

image

### CODE REVIEW

File: src/routes/InventoryTable.js [hunk 1]
```diff
@@ -168,6 +168,31 @@ const Inventory = ({
 
     const calculateSelected = () => selected ? selected.size : 0;
 
+    const isBulkRemoveFromGroupsEnabled = () => {
```
[the remark this answers is not in the document]
Comment (gkarat) [reply src/routes/InventoryTable.js:171]: I'd agree both have similar logic, but yet they are different. For the sake of readability, I would like to keep it as it is. @mkholjuraev

[the remark this answers is not in the document]
Comment (gkarat) [reply src/routes/InventoryTable.js:171]: Good point! I think a couple of everys can be merged together here...

[the remark this answers is not in the document]
Comment (gkarat) [reply src/routes/InventoryTable.js:171]: @mkholjuraev, check this out: https://github.com/RedHatInsights/insights-inventory-frontend/pull/1874/commits/44bb27619585be54fc201c5e201421d65638ef1e

**Comment by gkarat** [comment]
Looking at the CI tests results... there is an error in `test_navigate` which was said by Zhamilya that it can be unstable. Ignoring for now (the rest of the tests passes)

**Review by mkholjuraev — APPROVED** [review]
LGTM! I am able to bulk remove systems from a group and add multiple hosts

**Pull request merged**